flexibleengine/resource_flexibleengine_cce_cluster_v3.go

```diff
@@ -380,6 +397,13 @@ func resourceCCEClusterV3Read(d *schema.ResourceData, meta interface{}) error {
 	d.Set("authentication_mode", n.Spec.Authentication.Mode)
 	d.Set("security_group_id", n.Spec.HostNetwork.SecurityGroup)
 
+	// Set the custom can list
+	customSans := []string{}
+	for _, p := range n.Spec.CustomSan {
+		customSans = append(customSans, p)
+	}
+	d.Set("custom_san", customSans)
```

Comment:
we can set the field directly

```suggestion
	d.Set("custom_san", n.Spec.CustomSan)
```